### DISCUSSION TITLE
Regarding Game Backup Monitor and Windows 11

### DISCUSSION THREAD

**Discussion by MikeMaximus** [MikeMaximus/gbm#237] [General]
I will not be doing any development or testing with Game Backup Monitor on the upcoming Windows 11, at least for the foreseeable future.
 
I can't see there being any specific Windows 11 related problems, but it will not be officially supported.  

Thanks.

**Comment by NewRavenGer** [comment]
For what it's worth... using GBM with Windows 11 for a few days now and everything works as expected. But it totally makes sense that you wouldn't develop for a work-in-progress operating system not released, yet :)

**Reply by MikeMaximus** [reply]
Thanks, that's good to know. 👍 

I probably will throw it in a VM eventually just to make sure I can troubleshoot on it if i have to.
